### DISCUSSION TITLE
untitled

### DISCUSSION THREAD

**Discussion by laurentiucozma12** [drknzz/GitHub-Achievements#138] [Q&A] [answered]

**Comment by pu-raihan** [comment]


**Reply by laurentiucozma12** [reply]
done

**Reply by Maltered** [reply]
please, stars this one, I'm doing a competition with my friend and I need to win 🙏  https://github.com/Maltered/freemind-Project

**Comment by Maltered** [comment]
please, stars this one, I'm doing a competition with my friend and I need to win 🙏  https://github.com/Maltered/freemind-Project

**Reply by laurentiucozma12** [reply]
done

**Comment by ganochenkodg** [comment, marked as answer]
@laurentiucozma12  Done,pls star mine https://github.com/ganochenkodg/qdrant-operator

**Reply by laurentiucozma12** [reply]
done
